packages/web/app/pages/[orgId]/[projectId]/[targetId]/laboratory.tsx

```diff
@@ -1,54 +1,33 @@
 import { ReactElement } from 'react';
-import { createGraphiQLFetcher } from '@graphiql/toolkit'; // eslint-disable-line import/no-extraneous-dependencies
+import { createGraphiQLFetcher } from '@graphiql/toolkit';
 import { GraphiQL } from 'graphiql';
-import { useQuery } from 'urql';
 
 import { TargetLayout } from '@/components/layouts';
-import { noSchema, Spinner, Title } from '@/components/v2';
+import { Title } from '@/components/v2';
 import { HiveLogo } from '@/components/v2/icon';
-import { SchemasDocument } from '@/graphql';
 import { useRouteSelector } from '@/lib/hooks/use-route-selector';
 import 'graphiql/graphiql.css';
 
-const Page = () => {
+const Page = (): ReactElement => {
   const router = useRouteSelector();
-  const [schemasQuery] = useQuery({
-    query: SchemasDocument,
-    variables: {
-      selector: {
-        organization: router.organizationId,
-        project: router.projectId,
-        target: router.targetId,
-      },
-    },
-    requestPolicy: 'cache-and-network',
-  });
 
-  const hasSchemas = schemasQuery.data?.target.latestSchemaVersion?.schemas.nodes.length > 0;
   const endpoint = `${window.location.origin}/api/lab/${router.organizationId}/${router.projectId}/${router.targetId}`;
+
   return (
     <>
       <p className="mb-5 font-light text-gray-500">
         Experiment, mock and create live environment for your schema, without running any backend.
       </p>
-      {schemasQuery.fetching ? (
-        <Spinner className="mt-10" />
```

Comment:
I guess no need to show a spinner, in v1 graphiql showed an error if there is no connected schema